test/test_shared.cpp

```diff
@@ -2177,7 +2183,7 @@ TEST(Shared_WaitForChangeAfterOwnCommit)
 
 #endif
 
-TEST(Shared_InterprocessWaitForChange)
+NONCONCURRENT_TEST(Shared_InterprocessWaitForChange)
```

Comment:
Why are these nonconcurrent tests?

Reply:
You are right, they actually could run concurrently given that any concurrent test has a different path. I'll change it to a normal test.

Reply:
The test `Shared_InterprocessWaitForChange` needs to stay non-concurrent due to the nature of winfork and the use of a hard coded file path, there is a comment about this inside the test. I've changed the other instances though.